File: app/src/main/java/com/culture/santos/adapter/EventAdapter.java
package com.culture.santos.adapter;

import com.culture.santos.module.Event;
import com.google.android.gms.maps.model.Marker;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by Ricar on 03/09/2016.
 */
public class EventAdapter {
    private List<Event> events;

    private Marker auxMarker;

    public EventAdapter(){
        this.events = new ArrayList<>();
    }

    public Marker getMarker(){
        return auxMarker;
    }

    public void setMarker(Marker currentMarker){
        auxMarker = currentMarker;
    }

    public void addEvent(Event e){
        events.add(e);
    }

    public boolean cointainsEvent(Event e){
        return events.contains(e);
    }

    public Event getEvent(int position){
        return events.get(position);
    }

}

File: app/src/main/java/com/culture/santos/adapter/EventAdapter.kt
package com.culture.santos.adapter

import com.culture.santos.module.Event
import com.google.android.gms.maps.model.Marker

import java.util.ArrayList

/**
 * Created by Ricar on 03/09/2016.
 */
class EventAdapter {
    private val events: MutableList<Event>

    var marker: Marker? = null

    init {
        this.events = ArrayList()
    }

    fun addEvent(e: Event) {
        events.add(e)
    }

    fun cointainsEvent(e: Event): Boolean {
        return events.contains(e)
    }

    fun getEvent(position: Int): Event {
        return events[position]
    }

}
